.erb_lint.yml

```diff
@@ -0,0 +1,30 @@
+---
+EnableDefaultLinters: true
+linters:
+  ErbSafety:
+    enabled: false
+  RequireInputAutocomplete:
+    enabled: false
+  SelfClosingTag:
+    enabled: false
+  Rubocop:
+    enabled: true
+    rubocop_config:
+      inherit_from:
+        - .rubocop.yml
+      Layout/InitialIndentation:
+        Enabled: false
+      Layout/LineLength:
+        Enabled: false
+      Layout/TrailingEmptyLines:
+        Enabled: false
+      Layout/TrailingWhitespace:
+        Enabled: false
+      Naming/FileName:
+        Enabled: false
+      Style/FrozenStringLiteralComment:
+        Enabled: false
+      Lint/UselessAssignment:
+        Enabled: false
+      Rails/OutputSafety:
+        Enabled: false
```

Comment:
Would it be possible to enable these and see how many / which files are breaking them, and put in the ignore tags in the files. Because going forward it would be use to have these checks in place. 

Although I can see in this PR trailing empty lines have been removed? ...